modules/workspace.nix

```diff
@@ -99,14 +108,27 @@ in
         Allows you to set wallpaper slideshow. Needs a directory of your wallpapers and an interval length.
       '';
     };
+
+    wallpaperPictureOfTheDay = lib.mkOption {
+      type = lib.types.nullOr wallpaperPictureOfTheDayType;
+      default = null;
+      example = "apod";
```

Comment:
Wait... I don't think you can assign strings to submodules. Honestly, the submodule isn't even necessary and a simple enum type would do just fine

Reply:
Ah, yes that's completely true, I'll update the example.